definitions/tools/bcftools_merge.cwl

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+baseCommand: ["/opt/bcftools/bin/bcftools", "merge"]
+
+requirements:
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: DockerRequirement
+      dockerPull: "mgibio/bcftools-cwl:1.9"
+
+inputs:
+    force_merge:
+        type: boolean?
+        default: true
+        inputBinding:
+            position: 1
+            prefix: "--force-samples"
+        doc: "resolve duplicate sample names"
+    merge_method:
+        type: string?
```

Comment:
This could be an `enum` of the possible valid merge methods.

Reply:
I agree that's the best for this input